step-templates/File System - Copy File

```diff
@@ -0,0 +1,43 @@
+{
+  "Id": "ActionTemplates-46",
```

Comment:
`Id` should be a `GUID`, can be generated using this [site](https://www.uuidgenerator.net/guid)